kodi_addon_checker/check_addon.py

```diff
@@ -302,3 +306,19 @@ def _check_file_whitelist(report: Report, file_index, addon_path):
 def relative_path(file_path):
     path_to_print = file_path[len(REL_PATH):]
     return ".{}".format(path_to_print)
+
+
+def _check_complex_files(report: Report, addon_path, max_line_count):
+
+    addon_xml_path = os.path.join(addon_path, "addon.xml")
+    tree = xml.etree.ElementTree.parse(addon_xml_path).getroot()
+    complex_files = []
+
+    for i in tree.findall("extension"):
+        library = i.get("library")
+        if library:
+            filepath = os.path.join(addon_path, library)
+            num_line = sum(1 for line in open(filepath))
```

Comment:
please use `with` python statement so that `open(filepath)` is automatically closed:

```python
with open(filepath) as fp:
    num_line = sum(1 for line in fp)
```